src/worker.rs

```diff
@@ -97,13 +151,19 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
                         for attr in e.attributes() {
                             if let Ok(Attribute { value, key }) = attr {
                                 if decode_utf8(&key, &reader)? == "revision" {
-                                    revision = Some(decode_utf8(&value, &reader)?.parse::<i32>().map_err(|err| {
-                                        format!(
-                                            "Revision number parsing error at position {}: {:?}",
-                                            reader.buffer_position(),
-                                            err,
-                                        )
-                                    })?);
+                                    let unwrapped_revision =
```

Comment:
unwrapped is a bit overloaded. how about,
```suggestion
                                    let raw_revision =
```

or `revision_str`.